src/components/landing_page/UserAuthNav.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../style/breakpoints.scss';

```

Comment:
You don't need to import this if you are not using the breakpoints.
This means only if you do media queries to make different styling for mobile and desktop you'll need these.